tests/integ/test_df_to_pandas.py

```diff
@@ -77,8 +77,8 @@ def test_to_pandas_cast_integer(session, to_pandas_api, local_testing_mode):
     assert str(pandas_df.dtypes[2]) == "int32"
     assert str(pandas_df.dtypes[3]) == "int64"
     assert (
-        str(pandas_df.dtypes[4]) == "int8"
-    )  # When static type can possibly be greater than int64 max, use the actual value to infer the int type.
+        str(pandas_df.dtypes[4]) == "int64"
```

Comment:
is this considered a BCR?

Reply:
yeah, it is

Reply:
then we will have this put under the "BCR" section in the changelog?

Reply:
you are right. I'll update that.